docs/faq/templates.md

```diff
@@ -22,27 +22,27 @@
 		- https://blog.melbadry9.xyz/dangling-dns/aws/ddns-ec2-current-state
 		- https://blog.projectdiscovery.io/writing-nuclei-templates-for-wordpress-cves/
 
-??? info "How to run nuclei templates?"
+??? info "How do I run nuclei templates?"
 
-	Nuclei templates can be executed using template name or tags, using `-t`, `-tags` flag, respectively.
+	Nuclei templates can be executed using a template name or with tags, using `-templates` (`-t`) and `-tags` flag, respectively.
 
 	```
 	nuclei -tags cve -list target_urls.txt
 	```
 
-??? info "I want to contribute nuclei templates 😁"
+??? info "I want to contribute nuclei templates! 😁"
 
-	You are always welcome to share your nuclei template with the community. You can either open [GitHub issue](https://github.com/projectdiscovery/nuclei-templates/issues/new?assignees=&labels=nuclei-template&template=submit-template.md&title=%5Bnuclei-template%5D+template-name) with template details or open a Github [pull request](https://github.com/projectdiscovery/nuclei-templates/pulls) with your nuclei templates. If you don't have a GitHub account, you can also make use of [discord server](https://discord.gg/projectdiscovery) to share the template with us and we will make sure to add them to the project asap.
+	You are always welcome to share your nuclei template with the community. You can either open a [GitHub issue](https://github.com/projectdiscovery/nuclei-templates/issues/new?assignees=&labels=nuclei-template&template=submit-template.md&title=%5Bnuclei-template%5D+template-name) with the template details or open a Github [pull request](https://github.com/projectdiscovery/nuclei-templates/pulls) with your nuclei templates. If you don't have a GitHub account, you can also make use of the [discord server](https://discord.gg/projectdiscovery) to share the template with us.
 
 
 ??? warning "I'm getting false-positive results!"
-	Nuclei templates is a **community-contributed project**. We try our best to manually  review them before merging them into the template project. Still, there is a possibility that some templates with weak matchers will slip through the manual verification. This will produce false-positive results. **Templates are only as good as their matchers.**
+	Nuclei templates is a **community-contributed project**. We manually review templates before merging them into the  project. Still, there is a possibility that some templates with weak matchers will slip through the verification. This could produce false-positive results. **Templates are only as good as their matchers.**
```

Comment:
community contributed project, but our team manually reviews them before merging into the project. Making it clearer that the "we" does not refer to the community, but to a dedicated Project Discovery team.

Also there seems to be two spaces between "the" and "project".